TESTS/netsocket/tcp/tcpsocket_echotest_burst.cpp

```diff
@@ -73,6 +73,8 @@ void TCPSOCKET_ECHOTEST_BURST()
             if (recvd < 0) {
                 printf("[%02d] network error %d\n", i, recvd);
                 break;
+            } else if (recvd > BURST_SIZE) {
```

Comment:
Actually `} else if (recvd > bt_left) {`

Reply:
Good point, we can check not only the recv, but partly also the logic of the test.